### PR TITLE
Restored SearchBar Test for OrgList Page

### LINKED ISSUE #870: Restore Search Testing for OrgList.test.tsx
**Describe the bug**
We discovered that large sections of test code have been removed:
- https://github.com/PalisadoesFoundation/talawa-admin/pull/867#discussion_r1163969971

The sections of test code that need to be restored can be found here
- https://github.com/PalisadoesFoundation/talawa-admin/pull/826/files#diff-969071a6467f507443e6dded253411e5478202bb363f6b760369fffeba3a307dL459-L469

**To Reproduce**

See Above

**Expected behavior**

Tests for `src/screens/OrgList/OrgList.test.tsx` need to be restored to 100% coverage with search being thoroughly tested.

**Actual behavior**

See Above

**Screenshots**

N/A

**Additional details**

N/A

### PR DESCRIPTION
**Pull request by aviraldevv**
**What kind of change does this PR introduce?**
This PR restores the test for search-bar of the OrgList Page.

**Issue Number:**

Fixes #870 

**Did you add tests for your changes?**
Yes

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by aviraldevv** [comment]
@palisadoes 

I have restored the tests for search-bar successfully.

Screenshot_523

We can get this merged.
I will create a new issue to increase the code coverage for `OrgList` for someone else to work upon as I have very important exams coming week.

**Comment by palisadoes** [comment]
Create the remediation issue now.

**Review by palisadoes — APPROVED** [review]

File: src/screens/OrgList/OrgList.test.tsx [hunk 1]
```diff
@@ -233,11 +268,23 @@ describe('Organisation List Page', () => {
...
+
+    // Test that the search bar is case-insensitive
+    userEvent.clear(searchBar);
+    const search2 = 'aKaTsUkI';
+    expect(searchBar).toBeInTheDocument();
+    userEvent.type(screen.getByTestId(/searchByName/i), search2);
+
+    // Test that the search bar filters all organization if there are is no search passed
+    userEvent.clear(searchBar);
+    const search3 = '';
```
Comment (anshgoyalevil) [inline src/screens/OrgList/OrgList.test.tsx:285]: @aviraldevv 

Why have you used ```expect(searchBar).toBeInTheDocument();``` three times. 

The actual expect statements should check for the results after entering a query, and not the search bar itself.

Check the previous tests which were removed. They might give you an insight.

Comment (anshgoyalevil) [reply src/screens/OrgList/OrgList.test.tsx:285]: The following expect statements need to be added

```expect(container.textContent).toMatch(search1);```
.
.
etc.

**Comment by aviraldevv** [comment]
@palisadoes 
The tests are consistent with other tests of search-bar which have the same logic and also codecov has approved for the tests.
Search-bar is also functional for all the edge cases.

Screenshot_524

Screenshot_525

Screenshot_526

File: src/screens/OrgList/OrgList.test.tsx (hunk 1 above)
Comment (aviraldevv) [reply src/screens/OrgList/OrgList.test.tsx:285]: Thanks for you input buddy,
I researched on this and found out that container method at times cannot capture the text content of the filtered search results due to the way they are being rendered in the DOM. In that case, using the text content property of a specific element that contains the search results may be a more reliable approach which I have used.

**Comment by palisadoes** [comment]
@aviraldevv Please resubmit
